featurebyte/models/request_input.py

```diff
@@ -182,7 +219,22 @@ async def materialize(
                     ),
                 )
             )
-        if time_range_conditions:
+        if self.columns_rename_mapping:
+            # select only rows where renamed columns are not null
+            for col_name in self.columns_rename_mapping.values():
+                filter_conditions.append(
+                    expressions.Is(
+                        this=quoted_identifier(col_name),
+                        expression=expressions.Not(this=expressions.Null()),
+                    )
+                )
+        if filter_conditions:
+            # if the expression is a select * from table, we need to select the columns explicitly
+            if query_expr.expressions and query_expr.expressions[0].name == "*":
+                query_expr = select_and_rename_columns(
+                    query_expr, columns, self.columns_rename_mapping
+                )
```

Comment:
Is it possible that `select_and_rename_columns` is already called once above? If so will there be issue if the columns rename mapping is applied again?